index.src.html

```diff
@@ -1049,6 +1062,20 @@ <h3 id="interfaces-credential-manager">Credential Manager</h3>
         credential: The credential to be stored.
       </pre>
     </dd>
+    <dt><dfn method>bad()</dfn></dt>
+    <dd>
+      Notify the credential manager that an authentication or authorization via
+      the passed credential failed. This might be called because a credential
+      is expired.
+
+      When {{bad()}} is called, the user agement MAY offer the user to delete the
+      credential, MAY delete it silently or MAY apply different heuristics to
+      deal with the broken credential.
```

Comment:
This has no normative weight, as it's all just MAY. I'd suggest taking a more principled stance on what the user agent should do. What would you like Chrome to do when this is called?

Likewise, it might be helpful to add an implementation considerations section outlining why your principled stance is what it is. What kinds of things do user agents need to worry about when implementing this?